packet_service/types.lua

```diff
@@ -1689,38 +1689,39 @@ types.incoming[0x062] = struct({
 types.incoming[0x063] = multiple({
     base = struct({
         type            = {0x00, uint16},
+        size            = {0x02, uint16},
     }),
 
     lookups = {'type'},
 
     [0x02] = struct({
-        flags           = {0x02, data(7)}, -- The 3rd bit of the last byte is the flag that indicates whether or not you are xp capped (blue levels)
+        flags           = {0x00, data(7)}, -- The 3rd bit of the last byte is the flag that indicates whether or not you are xp capped (blue levels)
     }),
 
     [0x03] = struct({
-        flags1          = {0x02, data(2)}, -- Consistently D8 for me
-        flags2          = {0x04, data(2)}, -- Vary when I change species
-        flags3          = {0x06, data(2)}, -- Consistent across species
-        monstrosity_rank= {0x08, uint8}, -- 00 = Mon, 01 = NM, 02 = HM
-        infamy          = {0x0E, uint16},
-        instinct_flags  = {0x18, data(0x40)}, -- Bitpacked 2-bit values. 0 = no instincts from that species, 1 == first instinct, 2 == first and second instinct, 3 == first, second, and third instinct.
-        monster_levels  = {0x58, data(0x80)}, -- Mapped onto the item ID for these creatures. (00 doesn't exist, 01 is rabbit, 02 is behemoth, etc.)
+        flags1          = {0x00, data(2)}, -- Consistently D8 for me
+        flags2          = {0x02, data(2)}, -- Vary when I change species
+        flags3          = {0x04, data(2)}, -- Consistent across species
+        monstrosity_rank= {0x06, uint8}, -- 00 = Mon, 01 = NM, 02 = HM
```

Comment:
No space before = sign, topic discussed in discord.
it is an exception for files of this nature, long multi structure defining "types" files. 
the space is not require before the = when it would otherwise require the whole document to be realigned.